tests/test_prep/test_phase.py

```diff
@@ -61,6 +63,58 @@ def test_paganin_filter_padmean(data):
 
     assert_allclose(np.mean(filtered_data), -765.3401, rtol=eps)
     assert_allclose(np.min(filtered_data), -793.68787, rtol=eps)
+    # test a few other slices to ensure shifting etc is right
+    assert_allclose(
+        filtered_data[0, 50, 1:5],
+        [-785.60736, -786.20215, -786.7521, -787.25494],
+        rtol=eps,
+    )
+    assert_allclose(filtered_data[0, 50, 40:42], [-776.6436, -775.1906], rtol=eps)
+    assert_allclose(filtered_data[0, 60:63, 90], [-737.75104, -736.6097, -735.49884])
```

Comment:
Unfortunately, this test fails on my system, but increasing the epsilon value would make sure it'll pass.
```suggestion
    assert_allclose(filtered_data[0, 60:63, 90],
        [-737.75104, -736.6097, -735.49884], rtol=eps)
```